pkgs/os-specific/linux/open-iscsi/default.nix

```
@@ -29,8 +29,8 @@ stdenv.mkDerivation rec {
   meta = with stdenv.lib; {
     description = "A high performance, transport independent, multi-platform implementation of RFC3720";
     license = licenses.gpl2Plus;
```

```suggestion
    license = licenses.gpl2;
```
According to the file `COPYING` it does not mention any later versions of the GPL